src/todo/columns.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import defaultTodoColumns from "./config.json";
+import buildColumn from "../refapp-grid/column-builder";
+
+export default function getColumns() {
+  const getDoneActionColumn = () => {
+    return [];
+  };
+
+  const getTodoActionColumn = () => {
+    const fetchEncounterURL = (patientId, encounterId) => {
+      let baseUrl: string = `${location.origin}/openmrs/htmlformentryui/htmlform/viewEncounterWithHtmlForm.page?`;
```

Comment:
```suggestion
      const baseUrl: string = `/openmrs/htmlformentryui/htmlform/viewEncounterWithHtmlForm.page?`;
```

- use constant where ever it is possible.
- use relative path